packages/react-query/src/useSuspenseQueries.ts

```diff
@@ -109,6 +109,10 @@ export type SuspenseQueriesResults<
           : // Fallback
             Array<UseSuspenseQueryResult>
 
+/**
+ * This hook is wrapping useQueries of `@tanstack/react-query` v4 with default suspense option.
```

Comment:
I added the JSDoc for this function as it was missing.

Reply:
👍